src/ci/bin/testCentaurHoricromtalPapiV2.sh

```diff
@@ -40,6 +40,9 @@ cromwell::build::run_centaur \
     -p 100 \
     -e localdockertest \
     "${CROMWELL_BUILD_CENTAUR_TEST_ADDITIONAL_PARAMETERS:-""}" \
-    -d "${CROMWELL_BUILD_CENTAUR_TEST_DIRECTORY}"
+    -d "${CROMWELL_BUILD_CENTAUR_TEST_DIRECTORY}" \
+    -e relative_output_paths \
+    -e relative_output_paths_colliding \
+    -e standard_output_paths_colliding_prevented \
```

Comment:
Could these move up to join the existing `-e localdockertest` (or vice versa move that one down)?